src/main/resources/application.properties

```diff
@@ -23,4 +23,5 @@ management.endpoint.camelroutes.read-only=true
 camel.component.http.connect-timeout=5000
 camel.component.http.socket-timeout=30000
 camel.component.http.max-total-connections=1
-camel.component.http.connections-per-route=1
\ No newline at end of file
+camel.component.http.connections-per-route=1
+camel.springboot.shutdown-timeout=1
```

Comment:
bit quircky, but otherwise the System.exit on error will attempt a gracefull shutdown.